### ISSUE TITLE
Playing without audio on iOS 13.3.1

### ISSUE THREAD

**Issue by hansjjprins** [katspaugh/wavesurfer.js#1864]
### Wavesurfer.js version(s):
3.3.1 (2020-01-14)

### Browser and operating system version(s):
Safari on iOS 13.3.1

### Code needed to reproduce the issue:
```
var wavesurfer = WaveSurfer.create({
    container: '#waveform',
    waveColor: '#ccc',
    progressColor: '#000000',
    height: 40
});
wavesurfer.load('sample.mp3');
```

### Use behaviour needed to reproduce the issue:
Trying to play the audio does not create any errors and visually all seems to work. However, no sound can be heard. The same is true for examples that are supplied on the official Waversurfer website (https://wavesurfer-js.org/).

**Comment by kphurley** [comment]
Also running into this.  Affects both Chrome and Safari on iOS 13.3.1.  Could be related to this:  https://bugs.webkit.org/show_bug.cgi?id=203435

**Comment by thijstriemstra** [comment]
> EDIT: I got it working - user error (I had my phone's mute switch on :/)

so this can be closed?

**Comment by kphurley** [comment]
I’d say so yes.

**Comment by stuible** [comment]
I am also experiencing this issue on iOS Safari.  I can only hear the audio being played if I turn my phones ringer on.  If I have it on silent mode, I can hear normal audio but wavesurfer plays silence.  I am on iOS 13.3.1 as well/

**Comment by iamomiid** [comment]
Same on iOS 14

**Comment by entonbiba** [comment]
Chrome also has a auto play policy now that requires the user to initiate the audiocontext.
![image](https://user-images.githubusercontent.com/5193884/105644751-2f1bf000-5e65-11eb-9b91-1e3138f47553.png)

@iamomiid @stuible @thijstriemstra for web audio in ios it's silent on silent mode even on user interaction. ios considers the audio tag element as valid for initiating the sound on silent mode where web audio is not initiated by a audio element.

Only way to make it work would be to use html5 audio tag by creating a temporary audio element on page load that plays a silent audio file event 1 second length This would have to be initiated by the user clicking it so it can be a overlay on the play button just for ios devices that is clicked first. Once that is done web audio would work on silent mode.

I made it work here, tested on ios 14.2:
https://www.entonbiba.com/lab/ios2.html
```
<button onclick="iosAudio()">Play</button>
```
``` javascript
var audioios = document.createElement('audio');
audioios.id = 'audio-player-iosfix';
audioios.src = 'data:audio/mp3;base64,SUQzBAAAAAAAI1RTU0UAAAAPAAADTGF2ZjU2LjM2LjEwMAAAAAAAAAAAAAAA//OEAAAAAAAAAAAAAAAAAAAAAAAASW5mbwAAAA8AAAAEAAABIADAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDV1dXV1dXV1dXV1dXV1dXV1dXV1dXV1dXV6urq6urq6urq6urq6urq6urq6urq6urq6v////////////////////////////////8AAAAATGF2YzU2LjQxAAAAAAAAAAAAAAAAJAAAAAAAAAAAASDs90hvAAAAAAAAAAAAAAAAAAAA//MUZAAAAAGkAAAAAAAAA0gAAAAATEFN//MUZAMAAAGkAAAAAAAAA0gAAAAARTMu//MUZAYAAAGkAAAAAAAAA0gAAAAAOTku//MUZAkAAAGkAAAAAAAAA0gAAAAANVVV';
audioios.type = 'audio/mpeg';

// user initiates the play button on ios device which runs the following function
function iosAudio(){
    audioios.play();
    wavesurfer.play();
}

```

**Comment by entonbiba** [comment]
@thijstriemstra @katspaugh fyi I will open another **PR** to fix this ios silent issue once #2164 is merged. Will update the `.play()` method to run first before playing anything right before the interaction event is thrown.
https://github.com/katspaugh/wavesurfer.js/blob/master/src/wavesurfer.js#L809
![image](https://user-images.githubusercontent.com/5193884/105647583-a9a03c00-5e74-11eb-9c00-7a5019654644.png)


Since there's no way of knowing if a mobile device is in silent mode, running it before the `play()` should be fine, as it's an empty audio base64.

You may test it on your ios devices here:
https://www.entonbiba.com/lab/ios2.html
